app/views/browse/index.html.erb

```diff
@@ -1,6 +1,15 @@
 <% content_for :title, "All categories - GOV.UK" %>
 <% content_for :page_class, "browse" %>
 
+<% content_for :extra_headers do %>
+  <%= render partial: 'govuk_component/analytics_meta_tags', locals: { content_item: @content_item } %>
+<% end %>
+
+<% content_for :breadcrumbs do %>
+  <%= render partial: 'govuk_component/breadcrumbs', locals: @navigation_helpers.breadcrumbs %>
+  <div class="header-context"><!-- Deliberately empty. This will prevent Slimmer from adding the artefact-breadcrumb. --></div>
```

Comment:
It doesn't, but static adds a placeholder breadcrumb at https://github.com/alphagov/static/blob/master/app/views/root/_base.html.erb#L32-L39